lib/fluent/supervisor.rb

```diff
@@ -83,6 +83,7 @@ def after_run
 
     def run_rpc_server
       @rpc_server = RPC::Server.new(@rpc_endpoint, $log)
+      @reload_after_timestamp = Time.now
```

Comment:
Should use `Fluent::Clock.now` instead of `Time.now` as a timer.
`Time.now` is influenced by resetting system clock.